src/components/TopBar/TopBar.tsx

```diff
@@ -26,21 +80,124 @@ export const TopBar = () => {
     }
   }, [isLightTheme]);
 
-  const users = [
-    { id: 1, name: 'Peter Newton' },
-    { id: 2, name: 'Alice Nguyen' },
-    { id: 3, name: 'Paul Nasser' },
-    { id: 4, name: 'Anil Kapoor' },
-    { id: 5, name: 'Phil Murray' },
-    { id: 6, name: 'Bobby Fischer' },
-    { id: 7, name: 'Carla Rossi' },
-    { id: 8, name: 'Diane Chang' },
-    { id: 9, name: 'Evelyn Wood' },
-    { id: 10, name: 'Frank Black' },
-  ];
+  useEffect(() => {
```

Comment:
Due to the amount of code here, it justifies moving the Add Member and Remove Member into its own component otherwise TopBar is going to get too complicated to read